### PR TITLE
Initial Lobby backend implementation.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: resources/prosody-plugins/util.lib.lua [hunk 1]
```diff
@@ -172,8 +172,21 @@ function is_feature_allowed(session, feature)
     end
 end
 
+function starts_with(str, start)
+    return str:sub(1, #start) == start
+end
+-- healthcheck rooms in jicofo starts with a string '__jicofo-health-check'
```
Comment (paweldomas) [inline resources/prosody-plugins/util.lib.lua:178]: why do we need to act  differently for healthcheck rooms? or is it just  an optimization?

Comment (damencho) [reply resources/prosody-plugins/util.lib.lua:178]: Optimization, for example speaker stats print some messages when healthchecks were running ... Not that it is much as it is once every 10 secs.

Comment (damencho) [reply resources/prosody-plugins/util.lib.lua:178]: But for example speaker stats maybe triggering an http call after meeting has ended ... There are other examples like that, which should not execute on those rooms.

**Comment by damencho** [comment]
Lobby tests are work in progress.

**Pull request merged**